### ISSUE TITLE
Freeze on Exit with Pygame 2.1.2 Examples

### ISSUE THREAD

**Issue by Robin-Andrews** [pygame/pygame#3491]
I'm on Windows 10 using pygame 2.1.2 (SDL 2.0.18, Python 3.10.1).

I've tried pygame.examples.liquid and pygame.examples.stars, and both freeze when I click the exit icon, rather than exiting cleanly. This is not great for people wanting to use the examples to get a sense of good practice with writing Pygame code.

Any chance this could be addressed please?

**Comment by ankith26** [comment]
Hello! I'm on windows, and I can't reproduce the bug you describe.

I think it could actually be a bug of the IDE you are using (for instance, there is a weird known bug where pygame games run with IDLE have windows sticking around even after actually being closed. This is probably something having to do with the python process not exactly shutting down, and this is not pygames fault)

Could you test running the example directly from the windows commandline (like `py -m pygame.examples.liquid`) and see whether the issue persists?

**Comment by Starbuck5** [comment]
I can't reproduce the issue even in IDLE on 2.1.2.

And all the examples should be compliant with exiting on IDLE now (at least in the latest dev release).

**Comment by MyreMylar** [comment]
Yes, `pygame.quit()` was added to the old examples in this commit:
https://github.com/pygame/pygame/commit/8583d58221dd7b58d07a176b3a7214a28c814d95

My guess is that OP might have multiple versions of python & pygame installed, as this bug very was likely present in IDLE before 2.1.2. Either that or they have a more obscure IDE/python setup that is interacting badly.

@Robin-Andrews check the code you are running to see if it has a line like `pg.quit()` at the end. If it hasn't you are on a version before this was fixed, if it has you'll need to explain more about your dev setup.
